### ISSUE TITLE
rex_text for UITextField is property, but for UITextView SignalProducer

### ISSUE THREAD

**Issue by Igor-Palaguta** [neilpa/Rex#144]
I think for consistency we should have everywhere properties, or is there any reason for this?

**Comment by andersio** [comment]
Yeah, it looks like it should be a mutable & observable property for both of them. We can revisit these as we make progress towards ReactiveCocoa 5.0.
